### PR TITLE
Add missing labs cname

### PR DESCRIPTION
**Pull request by endakelly**
### Change description ###
Error in azure-platform-terraform pipeline resulting from missing dns record for shutter

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: environments/sandbox.yml [hunk 1]
```diff
@@ -268,3 +268,6 @@ cname:
   - name: "afdverify.labs-rhodrif-nodejs"
     ttl: 3600
     record: "afdverify.hmcts-sbox.azurefd.net"
+  - name: "cdnverify.labs-rhodrif-nodejs"
```
Comment (timja) [inline environments/sandbox.yml:271]: we should probably set `shutter: false` on those?

**Review by timja — APPROVED** [review]

File: environments/sandbox.yml [hunk 2]
```diff
@@ -264,7 +264,11 @@ cname:
     record: "sdshmcts-sbox.azurefd.net"
   - name: "labs-rhodrif-nodejs"
     ttl: 300
+    shutter: false
```
Comment (timja) [inline environments/sandbox.yml:267]: no i meant this goes in azure-platform-terraform so it doesn't create a shutter app

**Review by jordankainos — APPROVED** [review]
lgtm

**Comment by endakelly** [comment]
No longer needed. Shutter for the endpoint was removed in azure-platform-terraform which unblocked the pipeline

**Pull request closed without merging**